packages/react-button/src/components/Button/ButtonBase.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import { ButtonProps } from './Button.types';
+import { compose } from '@fluentui/react-compose';
+import { useButton } from './useButton';
+
+export const ButtonBase = compose<'button', ButtonProps, ButtonProps, {}, {}>(
+  (props, ref, composeOptions) => {
+    const { slots, slotProps } = useButton(props, composeOptions);
+
+    return (
+      <slots.root ref={ref} {...slotProps.root}>
+        {props.loading && <slots.loader {...slotProps.loader} />}
+        {props.icon && props.iconPosition !== 'after' && <slots.icon {...slotProps.icon} />}
+        {!props.iconOnly && props.content && <slots.content {...slotProps.content} />}
+        {props.icon && props.iconPosition === 'after' && <slots.icon {...slotProps.icon} />}
```

Comment:
Should all of this be inside a condition where `!props.loading`?

Reply:
Need to double check. I think so.

Reply:
v0 code still renders the icon even when loading is set.